tests/test_auto_detection.py

```diff
@@ -9,20 +10,22 @@
 from uvicorn.protocols.websockets.auto import AutoWebSocketsProtocol
 
 try:
-    import uvloop
+    importlib.import_module("uvloop")
+    expected_loop = "uvloop"
 except ImportError:  # pragma: no cover
-    uvloop = None
+    expected_loop = "asyncio"
```

Comment:
Open to discuss unused imports vs using importlib just to make sure package is installed.

Reply:
What was wrong with the import statement?

Reply:
The import statement does `import uvloop` just to see if uvloop is installed, so causes two issues:
- One is that we have unused import
- Second is that if package is not installed, we set `uvloop = None` which needs another `type: ignore` because module cannot be None.

Since this is just for purpose of checking package installation, I thought it makes sense to use importlib.